### ISSUE TITLE
Viewer current page number overflows toolbar in Firefox , but not Chrome

### ISSUE THREAD

**Issue by kaytuca** [mozilla/pdf.js#15084]
Attach (recommended) or Link to PDF file here: 
tracemonkey demo demonstrates  [clicking the link on github to the online viewer] 

Configuration:
- Tested in Chrome 102.0.5005.115 and in Firefox (101.0.1)  
- Operating system and its version: Win 10, 21H2 (OS Build 19044.1766)
- PDF.js version: the online demo version.  I also replicate using my separate implementation of the viewer on my website with other pdf files. My implementation of the viewer is from 2.14.305-legacy
- Is a browser extension: No

Steps to reproduce the problem:
1. Open any pdf in viewer.html in Chrome 
2.Open same pdf in viewer.html in Firefox OR open any pdf directly (not using the viewer) in Firefox (the same issue happens in Firefox with or without the viewer)

What is the expected behavior? (add screenshot)
The current page number box stays within the toolbar

What went wrong? (add screenshot)
The current page number box overflows the toolbar in Firefox, but not in Chrome

This is screenshot of the viewer tracemonkey demo in Firefox, and is also the same for any pdf opened directly in Firefox 
![tracemonkey in firefox](https://user-images.githubusercontent.com/76606355/175125573-2d505c22-ac57-4c8f-8f91-a3a02e527b3c.jpg)

This is a screenshot of the online viewer demo in Chrome
![Tracemonkey in chrome](https://user-images.githubusercontent.com/76606355/175125759-a3de55ef-b61a-429e-9e5b-05db593588c7.jpg)

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

I do not know if the overflow of the current page number is intentional, however it does not overflow in Chrome and I find the overflow a little odd.

**Comment by Snuffleupagus** [comment]
WFM, in Firefox Nightly 103. The only way that this can possibly happen is if you have, in the browser itself, previously enabled the "Zoom Text Only"-mode.[1]

Hence it's not really clear to me that this is something that we can necessarily "fix" here, because even if we set an explicit `height` for the input-element its text-contents will *still overflow* when that specific browser-zooming mode is being used (i.e. even if the `font-size` is given an `!important` attribute).

---
[1] Which can be done by pressing the <kbd>Alt</kbd> key, and then using the browser window menu as: `View -> Zoom -> Zoom Text Only`.

**Comment by kaytuca** [comment]
I had not set the Zoom Text Only mode.  But I did use your comment to track down the issue, which is related, and so I am responding via email to update you as to why this happened.

 

I had, in both browsers, increased the font size, though differently in both browsers because of how each works:

 

In Chrome > Settings > Appearance > Font size, I have chosen “very large”. This caused no problem for pdf.js

 

In Firefox > Settings > Fonts I have chosen Size 22. By itself, this causes no problems either.  However, in Firefox, I also had chosen (when the problem occurred) in the Advanced tab, a minimum font size of 22.  When I removed the minimum font size and set that to “none”, the problem disappeared.  

 

So I have fixed this, but on a pure accessibility and “useability” level, for people with eyesight not as good as others, it would be helpful if the fontsize of stuff on the toolbar could be configured or adjusted in easy ways or at least somewhat responsivee to the font size people are setting for the browser. I find the numbers in the toolbar very small and hard to read because my vision isn’t great, and this is on a desktop with a large screen.  Oddly, the layout for the page numbers on my ipad is easier to read because of how those page numbers are presented – more of an overlay and definitely larger. 

 

Thanks for your response.

 

Kay
